jiant/utils/tokenizers.py

```diff
@@ -17,6 +17,22 @@ def tokenize(self, sentence):
         raise NotImplementedError
 
 
+def select_tokenizer(args):
```

Comment:
nit: `select_tokenizer(tokenizer, input_module, **unused_kw):` and call as `select_tokenizer(**args.to_dict())`, and similarly with other functions like `select_pool_type` ?